quiz.py

```diff
@@ -11,14 +19,19 @@ def start_quiz(questions):
             print(f"({j}) {questions[i]['option'+str(j)]:30}", end='\t'); sleep(0.1)
             if not j%2: print()
 
-        try:
-            answer = int(input("\nEnter your option number: ")); sleep(0.2)
-            if not 0<answer<5:
-                print(chalk.yellow("Please enter a choice between 1 and 4\n")); sleep(0.4)
-                continue
-        except ValueError:
-            print(chalk.cyan('Invalid Choice\n')); sleep(0.4)
-            continue
+        # flag for controlling the loop
+        flag = False
+        while(flag==False):
```

Comment:
I also added this while loop for taking valid input input but @prkhrv  said his quiz won't give second chances.

Reply:
Yes, It's Not the Intention of a Quiz to give Second Chances.

Reply:
Oh. Okay no worries.